exercise7/index.js

```diff
@@ -0,0 +1,49 @@
+// This code will run when the page loads.
+window.addEventListener("load", () => {
+  // Get all the elements on the page that have click actions or will be changed as the game is played.
+  // For starters, I've added the message div and the reset button.
+  let rows = document.querySelectorAll(".row");
+  let listEntries = document.querySelectorAll(".listEntry");
+  let checkboxes = document.querySelectorAll("input[name=checkbox]");
```

Comment:
These three will always be empty lists when the page first loads, so it's not worth querying for them here. I see that these are lets so that they can be reassigned inside the listeners below. Instead, you should just assign them to `consts` within the listeners and remove them from here.

Reply:
Updated.